clients/rospy/src/rospy/impl/transport.py

```diff
@@ -93,15 +93,16 @@ def fileno(self):
     ## closed. Function will be passed this transport as an argument.
     ## @param self
     ## @param cleanup_callback fn(Transport): callback for when connection is closed
-    def set_cleanup_callback(self, cleanup_callback):
-        self.cleanup_cb = cleanup_callback
+    def add_cleanup_callback(self, cleanup_callback):
```

Comment:
Since the attribute is not explicitly private we have to assume someone is using it directly.